susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1087,6 +1087,22 @@
         'PDID' : [1946, 1712], 'BETAPDID' : [1928], 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15_X86_ARM,
         'DEST' : '/srv/www/htdocs/pub/repositories/sle/15/2/bootstrap/'
     },
+    'SLE-15-SP3-aarch64' : {
+        'PDID' : [2142, 1709], 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15_X86_ARM,
```

Comment:
You are missing the `BETAPDID`, for the Beta client tools. Check the SLE15SP2 definitions, you can use the same IDs.

Reply:
See https://github.com/uyuni-project/uyuni/pull/3261/files#diff-f55fddc0bdf6c50cd2d8405836ce20fc7167a07bb22b5828b9f973b063d2a83fL1074 for example